test/engine.spec.ts

```diff
@@ -65,7 +65,7 @@ describe('#buildFromArray', () => {
   })
 
   it('parsing error', () => {
-    const engine = HyperFormula.buildFromArray([['=A1B1']])
+    const engine = HyperFormula.buildFromArray([['=1A1']])
```

Comment:
In unit tests, the string `A1B1` was used to cause a parsing error. Now, it causes no error because it is a valid named expression name. The string `1A1` is invalid because named expression names cannot start with a digit.